Stripe/STPCardParams.swift

```diff
@@ -0,0 +1,195 @@
+//
+//  STPCardParams.swift
+//  Stripe
+//
+//  Created by Jack Flintermann on 10/4/15.
+//  Copyright © 2015 Stripe, Inc. All rights reserved.
+//
+
+import Foundation
+
+/// Representation of a user's credit card details. You can assemble these with
+/// information that your user enters and then create Stripe tokens with them using
+/// an STPAPIClient.
+/// - seealso: https://stripe.com/docs/api#cards
+public class STPCardParams: NSObject, STPFormEncodable, NSCopying {
+  var additionalAPIParameters: [AnyHashable: Any] = [:]
+
+  /// The card's number.
+  @objc public var number: String?
+
+  /// The last 4 digits of the card's number, if it's been set, otherwise nil.
+  @objc
+  public func last4() -> String? {
+    if number != nil && (number?.count ?? 0) >= 4 {
+      return (number as NSString?)?.substring(from: (number?.count ?? 0) - 4)
+    } else {
+      return nil
+    }
+  }
+
+  /// The card's expiration month.
+  @objc public var expMonth: UInt = 0
+  /// The card's expiration year.
+  @objc public var expYear: UInt = 0
+  /// The card's security code, found on the back.
+  @objc public var cvc: String?
+  /// The cardholder's name.
+  /// @note Changing this property will also changing the name of the
+  /// param's `address` property.
+
```

Comment:
this file has many instances of empty lines after the doc strings. Are those okay?

Reply:
Yeah, jazzy/xcode will still associate them correctly. I'll clean these ones up though, it's easier to read that way